### ISSUE TITLE
A11Y changes for the carousel

### ISSUE THREAD

**Issue by mattpen** [phetsims/website-meteor#75]
- [x] Turn on auto rotation (5-7s)
- [ ] Add TAB accessible button for stopping auto rotate
- [ ] Stop auto-rotate on mouse-over (mostly handled by slick, but auto-rotate still happens when hovering over prev/next and dots)
- [ ] Stop auto-rotate on focus (supposed to be handled by slick, see `pauseOnFocus`, but it's not)
- [x] Prev/next/dots need more obvious focus highlight
- [ ] Implement TAB order -- Rotation Control -> Prev -> Next -> Dots -> Slide
- [ ] Implement arrow key nav for the dots
- [ ] Make sure all focusable elements have an appropriate label if necessary
- [ ] Only the currently visible slide should be accessible with TAB, add management of which slide is focusable with TAB

**Comment by chrisklus** [comment]
The carousel has been omitted, closing.
